copydlls.sh

```diff
@@ -233,7 +239,7 @@ copystuff() {
     find "${TARGETDIR}/lib/python2.7/" -name "*.pyo" -type f -delete
 
     echo Copying ssl/certs/ca-bundle.crt...
-    cp "$MSYSDIR/ssl/certs/ca-bundle.crt" "$TARGETDIR/ssl/certs"
+    cp "$MSYSDIR/ssl/certs/ca-bundle.crt" "$TARGETDIR/ssl/certs/"
```

Comment:
Because this is broken. It tries to copy `certs/ca-bundle.crt` into *file* named `certs`. It even fails when there is already a directory named `certs`.

But I will split this and some other changes into separate commits.

Reply:
Split off